### PR TITLE
Reference to [Online Python Compiler]

### PR DESCRIPTION
**Pull request by michael-freidgeim-webjet**
## Description

Tip if someone do not have python in the path

## Motivation and Context

Link to [Online Python Compiler]

## How Has This Been Tested?

Just documentation

## Checklist:

- [ X] My change requires a change to the documentation or CHANGELOG.
- [ X] I have updated the documentation/CHANGELOG accordingly.
- [ ] I have created a feature (non-master) branch for my PR.

### CODE REVIEW

File: docs/docs/configuration/overview.md [hunk 1]
```diff
@@ -7,7 +7,8 @@ title: Overview
 
 ### Generating a Cookie Secret
 
-To generate a strong cookie secret use `python -c 'import os,base64; print(base64.urlsafe_b64encode(os.urandom(16)).decode())'`
+To generate a strong cookie secret use `python -c 'import os,base64; print(base64.urlsafe_b64encode(os.urandom(16)).decode())'`  
+You can use [Online Python Compiler](https://www.tutorialspoint.com/execute_python_online.php)
```
Comment (JoelSpeed) [inline docs/docs/configuration/overview.md:11]: This is a great suggestion, just want to expand the sentence a little as below
```suggestion
You can use this [Online Python Compiler](https://www.tutorialspoint.com/execute_python_online.php) if you don't have python available on your own system.
```

**Review by JoelSpeed — COMMENTED** [review]
Would you be able to copy this new sentence to the two sets of versioned docs we have too? This allows those looking at the older docs to also get this new tip

**Comment by NickMeves** [comment]
I'm going to close this PR for security reasons as documented on the other PR that has this commit: https://github.com/oauth2-proxy/oauth2-proxy/pull/1094#discussion_r593521916

Untrusted online webapps shouldn't be used for secret generation.

**Comment by MNF** [comment]
I understand your concern in general, but it sounds a bit paranoiac to worry that a generic online compiler site will monitor random compiled function output and try to substitute it on any site that a user can setup with this cookie secret. 
Probably brute force attacks https://martinfowler.com/articles/session-secret.html#WhatIsASessionSecret
would be more successful

**Comment by NickMeves** [comment]
Trust me, I've seen some crazy stuff in my career for how advanced nation state APT hackers will compromise high value targets -- user tracking on a web tool doesn't even rate as that rare 😅 

At the end of the day, this field is a 16, 24, or 32 character password -- just type in some garbage or use a password generator.

If missing python is a concern (Windows?) - I'd accept a powershell alternative that uses a cryptographically secure random number generator.

**Comment by JoelSpeed** [comment]
Perhaps we can make it a tab group that has several examples, python, bash, powershell, ruby maybe?

**Comment by MNF** [comment]
I think, to add what @NickMeves already mentioned will be sufficient.

If you don't have python available on your own system, you can use any (not online)  password generator or even type some random sequence of characters. Ensure that the length Is either 16, 24, or 32 characters

**Comment by JoelSpeed** [comment]
I can add that, but I quickly hacked together #1108 which might help some users. Out of interest, what system are you using that doesn't have python, is it windows?

**Comment by MNF** [comment]
Yes, Windows. I could install python, but found it’s quicker to run the online compiler.
But now Nick convinced me, that it is not a good idea, at least you should not recommend it.

**Pull request closed without merging**